<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDB</class>
 <widget class="QWidget" name="PDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PDB ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewPDB3D">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View PDB in 3D</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditFasta">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>FASTA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDownloadPDB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pdb-tools</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Atom: CA, CB or CoM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPDB_ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPDB_filename">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chains of interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAtom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditChainsOfInterest">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllowJumps">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Allow jumps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattice type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cubic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattice distance:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxLatticeDistance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAverageDistancePDB">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Average from PDB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Discretize side chains as well</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDiscretizeSideChains">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDiscretizedPDB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>330</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>310</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Discretized PDB</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxListChains">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chains read from the PDB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileMergedChains">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMergeChains">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Merge chains into file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWorkingDirectory">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Working directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Python command (if not in the PATH)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPythonCommand">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileOutputLatfit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDiscretize">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Discretize into file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewDiscretized">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View PDB+ discretized in 3D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewReceptorStructures">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View receptor structures</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditMultiChainStructure">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConvertLattice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Convert into lattice into file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Multi-chain structure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileConvertedLattice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>latfit</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>21</width>
     <height>181</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRandomlyPicked">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>randmoly picked</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>number to show:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNumberReceptorsToShow">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ymir</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>21</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>size receptors</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min interact</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSizeReceptors">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMinInteract">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonView3DLatfitOutput">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View 3D Latfit Output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIterate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>iterate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDeInsert">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DeInsert</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxnKeep">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nKeep struct per iteration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
